shell/browser/api/electron_api_web_contents.cc

```diff
@@ -486,9 +486,13 @@ void OnCapturePageDone(gin_helper::Promise<gfx::Image> promise,
                        const SkBitmap& bitmap) {
   auto ui_task_runner = content::GetUIThreadTaskRunner({});
   if (!ui_task_runner->RunsTasksInCurrentSequence()) {
-    ui_task_runner->PostTask(
-        FROM_HERE, base::BindOnce(&OnCapturePageDone, std::move(promise),
-                                  std::move(capture_handle), bitmap));
+    if (!ui_task_runner->PostTask(
+            FROM_HERE, base::BindOnce(&OnCapturePageDone, std::move(promise),
+                                      std::move(capture_handle), bitmap))) {
+      // When `PostTask` returns `false` the task definitely will not run.
+      promise.RejectWithErrorMessage("Unknown error");
```

Comment:
Let's put something a little more descriptive here - `Failed to capture page` perhaps?

Reply:
Changed the error message [here](https://github.com/electron/electron/pull/41309/commits/c12528259dfd8d359198a0a8c447b2d225624ff4)